src/main/java/org/testeditor/fixture/web/WebDriverFixture.java

```diff
@@ -837,4 +851,46 @@ private String join(Keys[] keys) {
                  .map(n -> n.name())
                  .collect(Collectors.joining("\", \""));
     }
+
+    /**
+     * Checks if a specified text is on a web site. Even just typed Text-Artifacts 
+     * which are currently inserted into the DOM.  
+     * 
+     * @param textTobeFound
+     * @return true if searched text is found on a HTML Page and in DOM 
+     */
+    @FixtureMethod
+    public boolean isTextOnPage(String textTobeFound) {
+        boolean textOnpage = false;
+        List<WebElement> elements = driver.findElements(By.xpath("//*"));
+        // First looking in the page source for fix Web Elements
+        if (checkForConstantWebElements(textTobeFound)) {
+            return true;
+        }
+        // Now check for dynamic Web Elements in DOM 
+        textOnpage = checkForDynamicWebElements(textTobeFound, textOnpage, elements);
+        return textOnpage;
+    }
+
+    private boolean checkForConstantWebElements(String textTobeFound) {
+        String pageSource = driver.getPageSource();
+        return pageSource.contains(textTobeFound); 
+    }
+
+    private boolean checkForDynamicWebElements(String textTobeFound, boolean textOnpage, List<WebElement> elements) {
+        for (WebElement element :elements) {
+            // String tagName = element.getTagName();
+            String text = element.getText();
```

Comment:
looks like duplicate code for getting text via getText and via getAttribtue (see readValue)